bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+set -e
+
+if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
+	if [ -n "$MYSQL_PORT_3306_TCP" ]; then
```

Comment:
I'm pretty sure we want to get this from Consul, and not support injection via env vars.